### PR TITLE
Getting Started docs improvements

### LINKED ISSUE #162: Show system requirements on the docs
Currently it is just hidden on the unsupported browser page. I suggest we expose this further on the documentation.

### PR DESCRIPTION
**Pull request by Hans5958**
Closes #162

### CODE REVIEW

**Comment by Hans5958** [comment]
All docs (excl some) are translated, so yeah

**Comment by Hans5958** [comment]
@WorldLanguages What do you think?

File: content/docs/getting-started/quick-start.md [hunk 1]
```diff
@@ -7,22 +7,33 @@ aliases:
 
 ## Installing
 
-Let's get started by installing Scratch Addons. Scratch Addons is available in both Chrome Web Store and Add-ons for Firefox.
+Let's get started by installing Scratch Addons.
+
+| Store | Install | Supported browsers |
+| - | - | - |
+| Chrome Web Store | [![Install for Chrome Web Store](https://img.shields.io/chrome-web-store/v/fbeffbjdlemaoicjdapfpikkikjoneco?style=flat-square&logo=google-chrome&logoColor=white&label=install&color=4285F4)](https://chrome.google.com/webstore/detail/fbeffbjdlemaoicjdapfpikkikjoneco) | Google Chrome and other Chromium-based browser
```
Comment (WorldLanguages) [inline content/docs/getting-started/quick-start.md:14]: > other Chromium-based browser

Should be "others" and "browsers"

File: content/docs/getting-started/quick-start.md [hunk 2]
```diff
@@ -7,22 +7,33 @@ aliases:
...
 ## Getting started
 
 To get started, you need to access the settings page of the addon to see the functions of it.
 
-<!-- TODO: Add pictures -->
-
-1. Locate Scratch Addons on the toolbar and click it.  
+1. Locate Scratch Addons on the toolbar and click it.
+   
    Usually, it is on the top-right corner of your browser. For Google Chrome users, if you are not browsing Scratch, you need to click the puzzle icon to find the extension icon.
```
Comment (WorldLanguages) [inline content/docs/getting-started/quick-start.md:27]: > if you are not browsing Scratch

I know this has been here for a while, but I don't think this is accurate? The puzzle icon needs to be clicked even if not browsing Scratch.

File: content/docs/getting-started/quick-start.md [hunk 3]
```diff
@@ -7,22 +7,33 @@ aliases:
...
 2. Press the gear icon to go to the Settings page.
 
-3. Feel free to look through the addons list and turn on/off whatever you want.  
-   You can also configure each of the addons on the bottom section of each addons.
\ No newline at end of file
+   ![](/assets/img/getting-started/step-2.png)
+
+3. Feel free to look through the addons list and turn on/off whatever you want.
+   
+   You can also configure each of the addons on the bottom section of each addons.
```
Comment (WorldLanguages) [inline content/docs/getting-started/quick-start.md:37]: > You can also configure each of the addons on the bottom section of each addons.

This sentence is taken from somewhere else I think, but it's phrased weirdly

Comment (WorldLanguages) [reply content/docs/getting-started/quick-start.md:37]: Actually I see it's been on this doc page forever so we could change it later too.

Comment (WorldLanguages) [reply content/docs/getting-started/quick-start.md:37]: Yeah, this is #171

File: content/docs/getting-started/quick-start.md (hunk 1 above)
Comment (GrahamSH-LLK) [reply content/docs/getting-started/quick-start.md]: I'm pretty sure it should be "other" and "browsers"

Comment (WorldLanguages) [reply content/docs/getting-started/quick-start.md]: Yeah, my bad

**Comment by Hans5958** [comment]
I may also edit the main repo's README to add the information (it needs update anyway since Edge)

**Review by Wjplatformer — APPROVED** [review]

**Pull request merged**